infra/cmake/packages/TensorFlowGpuSource/patch_for_gpu_cl_build.patch

```diff
@@ -0,0 +1,292 @@
+diff --git a/tensorflow/lite/delegates/gpu/api.h b/tensorflow/lite/delegates/gpu/api.h
+index 7892d0ce..fae4fb69 100644
+--- a/tensorflow/lite/delegates/gpu/api.h
++++ b/tensorflow/lite/delegates/gpu/api.h
+@@ -43,11 +43,18 @@ limitations under the License.
+ #include "tensorflow/lite/delegates/gpu/common/data_type.h"
+ #include "tensorflow/lite/delegates/gpu/common/status.h"
+ #include "tensorflow/lite/delegates/gpu/common/util.h"
++
++#ifdef TFLITE_GPU_LIB_FIX
```

Comment:
@hseok-oh 
Actually `#ifdef TFLITE_GPU_LIB_FIX` means `if 0`.
Because our final goal is that use original `tflite.so` for gpu delegation.
If we ready, patch_for_gpu_cl_build.patch will be removed also.

So. this PR is a middle step for that.

Reply:
@hseok-oh 
Actually `#ifdef TFLITE_GPU_LIB_FIX` means `if 0`.
Because our final goal is that use original codes of tensorflow-lite for gpu delegation.
If we ready, patch_for_gpu_cl_build.patch will be removed also.

So please look this PR  which it is a middle step for that.

Reply:
I see.